src/Core/Checkout/Customer/SalesChannel/RemoveWishlistProductRoute.php

```diff
@@ -60,21 +60,31 @@ public function getDecorated(): AbstractRemoveWishlistProductRoute
      * @Since("6.3.4.0")
      * @OA\Delete(
      *      path="/customer/wishlist/delete/{productId}",
-     *      summary="Delete customer wishlist",
+     *      summary="Remove a prduct from a wishlist",
+     *      description="Removes a product from a customer's wishlist.
+
+     **Important constraints**
+
+     * Anonymous (not logged-in) customers can not have wishlists.
+     * The wishlist feature has to be activated.",
      *      operationId="deleteProductOnWishlist",
      *      tags={"Store API", "Wishlist"},
      *      @OA\Parameter(
      *        name="productId",
      *        in="path",
-     *        description="Product Id",
-     *        @OA\Schema(type="string"),
+     *        description="Product id",
+     *        @OA\Schema(type="string", format="uuid"),
      *        required=true
      *      ),
      *      @OA\Response(
      *          response="200",
-     *          description="Success",
+     *          description="Returns a success response.",
```

Comment:
What does a success response indicate? Same for failure. Why would the request fail? Product not in wishlist, or other reasons? Just saying `Returns a success response` is not really helpful to API users.